### PR TITLE
feat: add network id card

### PR DESCRIPTION
**Pull request by fhildeb**
This PR is adding a card to get the network ID using the Ethereum API instead of web3.js.

Bildschirmfoto 2022-10-07 um 12 39 29

Internal ticket: https://app.clickup.com/t/2znnbyb

### CODE REVIEW

**Review by Hugoo — COMMENTED** [review]
V model is prob not what you need - just use a state variable and display it - see how it is done in the other components

**Review by dzbo — CHANGES_REQUESTED** [review]
- please use `ref` and `model` binding to get value (check how other cards are done)
- show result in `Notifications` component, also potential error if any

File: src/components/endpoints/GetNetId.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+</script>
+
+<template>
+  <div class="tile is-4 is-parent">
+    <div class="tile is-child box">
+      <p class="is-size-5 has-text-weight-bold mb-4">Get Network ID</p>
+      <div class="field">
+        <div class="is-one-third mb-4">
+          <button
+            :class="`button is-primary is-rounded mt-4 `"
```
Comment (Hugoo) [inline src/components/endpoints/GetNetId.vue:28]:
```suggestion
            class="button is-primary is-rounded mt-4"
```

The `:` symbol is used to pass JS variables, so here you don't need it, and you dont need the ` either

https://laracasts.com/discuss/channels/vue/when-to-use-colon-when-passing-data-to-props

Comment (fhildeb) [reply src/components/endpoints/GetNetId.vue:28]: true, i should've recognized that before pushing

**Review by Hugoo — APPROVED** [review]
lgtm - @dzbo I don;t this this is worth adding tests.
I'm fine to merge.

**Review by dzbo — APPROVED** [review]

**Pull request merged**